### DISCUSSION TITLE
Finish workcation course?

### DISCUSSION THREAD

**Discussion by fimbault** [tailwindlabs/tailwindcss#2442] [General]
Hi, 

First of all thanks for the great work. People in my team have enjoyed https://tailwindcss.com/course, but it's not finished.
Is there a plan to finish the course?

Thanks

**Comment by adamwathan** [comment]
Hey! Yeah I am planning to re-record the entire thing later this year when we release Tailwind 2.0 so that it's up to date. Seems silly to invest in it right now as we're only a few months away from the next version 👍🏻

**Reply by fimbault** [reply]
Thanks, makes sense. Looking forward to it ;-)
